middleware/faq.rst

```diff
@@ -436,3 +436,77 @@ What do I do if a query method/command or pipetask graph generation is slow?
+      HSC/raw/RC2/9813                                   TAGGED
+      HSC/calib/gen2/20180117                            CALIBRATION
+      HSC/calib/DM-28636                                 CALIBRATION
+      HSC/calib/gen2/20180117/unbounded                  RUN
+      HSC/calib/DM-28636/unbounded                       RUN
+      HSC/masks/s18a                                     RUN
+      HSC/fgcmcal/lut/RC2/DM-28636                       RUN
+      refcats/DM-28636                                   RUN
+      skymaps                                            RUN
+    u/jbosch/DM-30649/20210614T191615Z                   RUN
+
+The RUN collections that directly hold the datasets are what we want to remove in order to free up space, but we have to start by deleting the `~CollectionType.CHAINED` collections that hold them first:
+
+.. code:: sh
+
+    $ butler remove-collection /repo/main u/jbosch/DM-30649 --no-confirm
+
+You can drop the ``--no-confirm`` option if you like.
+If you're only deleting one collection at a time, it doesn't tell you anything new.
+
+If you don't want to remove the `~CollectionType.CHAINED` collection - you just want to remove the `~CollectionType.RUN` collection from it - you can instead do
+
+    $ butler collection-chain /repo/main --remove u/jbosch/DM-30649 u/jbosch/DM-20210614T191615Z
+
+Or, if you know the `~CollectionType.RUN` is the first one in the chain,
+
+    $ butler collection-chain /repo/main --pop u/jbosch/DM-30649
+
+In any case, once the `~CollectionType.CHAINED` collection is out of the way, we can delete the `~CollectionType.RUN` collections that start with the same prefix using a glob pattern:
+
+.. code:: sh
+
+    $ butler remove-runs /repo/main u/jbosch/DM-30649/*
+    The following RUN collections will be removed:
+    u/jbosch/DM-30649/20210614T191615Z
+    The following datasets will be removed:
+    calexp(18222), calexpBackground(18222), calexp_camera(168), calibrate_config(1), calibrate_metadata(18222), characterizeImage_config(1), characterizeImage_metadata(18231), consolidateSourceTable_config(1), consolidateVisitSummary_config(1), consolidateVisitSummary_metadata(168), fgcmBuildStarsTable_config(1), fgcmFitCycle_config(1), fgcmOutputProducts_config(1), icExp(18231), icExpBackground(18231), icSrc(18231), icSrc_schema(1), isr_config(1), isr_metadata(18232), postISRCCD(18232), skyCorr(17304), skyCorr_config(1), skyCorr_metadata(168), source(18222), src(18222), srcMatch(18222), srcMatchFull(18222), src_schema(1), transformSourceTable_config(1), visitSummary(168), writeSourceTable_config(1), writeSourceTable_metadata(18222)
+    Continue? [y/N]: y
+    Removed collections
+
+Here we've left the default confirmation behavior on because we used a glob, just to be safe.
+You can write one or more full RUN collection names explicitly, too, and that's what you'll need to do if you didn't follow the naming convention well enough for a glob to work.
```

Comment:
Is something like this clearer?
```
You can write one or more full RUN collection names explicitly too by adding them individually separated by commas, and that's...
```

Reply:
You can use wildcards as well.

Reply:
I see that was a response to a glob comment. Anyhow, I'm not sure why `butler remove-runs` has not been set up to take multiple space separated run names either. It seems like it should just let you specify more than one with spaces like a standard command line.

Reply:
Ah, maybe it can take multiple space-separated runs too? I've always just added them comma-separated, so wanted to make those instructions clear here. If space-separated is fine too, the above can be amended to `individually separated by commas or whitespace`.

Reply:
Looks like spaces are not supported, but I agree with Tim that they should be (and it's what I expected to work; wouldn't have guessed commas).